tests/app/data/test_schema_id_regex.py

```diff
@@ -14,7 +13,7 @@ def create_schema_with_id(schema_id='answer'):
     :param schema_id: The Id to use for the answer.
     :return: The JSON file with the Id swapped for schema_id
     """
-    json_file = open(os.path.join(settings.EQ_SCHEMA_DIRECTORY, "test_percentage.json"))
+    json_file = open(_get_schema_file_path("test_percentage.json", language_code='en'))
```

Comment:
If we're using `_get_schema_file_path` outside of `app.utilities.schema` we should make them public (remove the _ from the start of the method name)